src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -14,6 +14,7 @@
       "description": "The set of runs contained in this log file.",
       "type": "array",
       "minItems": 1,
+      "uniqueItems": true,
```

Comment:
I don't think you can say this, given the preponderance of optional values. Unless a run contains start time and end time, you could easily imagine two runs with the same result (empty, for example).